deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,202 @@
+; Shows a lot of debug information while compiling
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+;Include Modern UI
+
+    !include "MUI2.nsh"
+    Var _JELLYFINVERSION_
+    Var _DEFAULTEMBYDATADIR_
+    Var _JELLYFINDATADIR_
+    Var _SERVICEINSTALLED_
+;--------------------------------
+;General
+
+; Align installer version with jellyfin.dll version
+    !getdllversion "$%InstallLocation%\jellyfin.dll" ver_
+    !echo "jellyfin.dll version is ${ver_1}.${ver_2}.${ver_3}.${ver_4}" ;!echo will print it while building
+
+    Name "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    OutFile "Jellyfin.Installer.${ver_1}.${ver_2}.${ver_3}.${ver_4}.exe"
+    BrandingText "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4} Installer"
+
+; installer attributes
+    VIProductVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIFileVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIAddVersionKey "ProductName" "Jellyfin"
+    VIAddVersionKey "FileVersion" "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+  
+  ;Default installation folder
+    InstallDir "$APPDATA\Jellyfin"
+  
+  ;Get installation folder from registry if available
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallLocation"
+  
+  ;Request application privileges for Windows Vista
+    RequestExecutionLevel admin
+    CRCCheck on	    
+    !define MUI_ABORTWARNING
+
+;--------------------------------
+;Pages
+
+;TODO
+;find a license to displayed before installer is started
+;    !insertmacro MUI_PAGE_LICENSE "<PATH TO LICENSE TXT FILE"
+    !insertmacro MUI_PAGE_COMPONENTS
+    !insertmacro MUI_PAGE_DIRECTORY
+
+; Custom Directory page to ask for Emby Library location in case its needed
```

Comment:
I don't think we want to support copying Emby library and config over like this, as we've seen a large amount of issues related to migrated Emby libraries, where the fix has usually been to just start fresh.

Reply:
Commented out the section to display the Emby migration page. 
Leaving the code in there if we find a better way of doing it later.